engine/src/main/java/org/terasology/rendering/cameras/OrthographicCamera.java

```diff
@@ -89,7 +89,7 @@ public void updateMatrices(float fov) {
             return;
         }
 
-        projectionMatrix = MatrixUtils.createOrthogonalProjectionMatrix(left, right, top, bottom, zNear, zFar);
+        projectionMatrix = new Matrix4f().ortho(left,right,top,bottom,zNear,zFar).transpose();
```

Comment:
The top and bottom parameters need to be switched.
```suggestion
        projectionMatrix = new Matrix4f().ortho(left,right,bottom,top,zNear,zFar).transpose();
```